src/scaffoldmaker/annotation/body_terms.py

```diff
@@ -0,0 +1,30 @@
+"""
+Common resource for body annotation terms.
+"""
+
+# convention: preferred name, preferred id, followed by any other ids and alternative names
+body_terms = [
+    ( "abdomen", "UBERON:0000916", "ILX:0725977" ),
+    ( "thorax", "ILX:0742178" ),
+    ( "neck", "UBERON:0000974", "ILX:0733967" ),
+    ( "head", "UBERON:0000033", "ILX:0104909" ),
+    ( "skin epidermis", "UBERON:0001003", "ILX:0728574" ),
+    ( "diaphragm", "UBERON:0001103", "ILX:0103194" ),
+    ( "spinal cord", "UBERON:0002240", "ILX:0110909" ),
+    ( "body", "UBERON:0000468", "ILX:0101370" ),
+    ( "core", "" ),
+    ( "non core", "" ),
+    ( "core boundary", "" ),
+    ( "non core first layer", "" ),
```

Comment:
This 'non core first layer' should not be an annotation group. It looks like you could use a Zinc FieldGroup FieldMeshGroup independently of the Annotations to do your stuff, then remove it after use?

Reply:
Done